package.json

```diff
@@ -34,18 +34,27 @@
     "postsetup": "npm run build:dll",
     "clean": "shjs ./internals/scripts/clean.js",
     "clean:all": "npm run analyze:clean && npm run test:clean && npm run build:clean",
-    "generate": "plop --plopfile internals/generators/index.js",
+    "generate": "plop --plopfile internals/generators/index.js && npm run prettier:js",
     "lint": "npm run lint:js && npm run lint:css",
     "lint:eslint": "eslint --ignore-path .gitignore --ignore-pattern internals/scripts",
     "lint:js": "npm run lint:eslint -- . ",
-    "lint:css": "stylelint \"./app/**/*.js\"",
+    "lint:css": "stylelint './app/**/*.js'",
     "lint:staged": "lint-staged",
     "pretest": "npm run test:clean && npm run lint",
+    "prettier:js": "prettier --write './{app, internals}/**/*.js'",
```

Comment:
How about all the folder except `build`

Reply:
we can even format `package.json` and markdowns

Reply:
Good idea.